faaserrors/errors.go

```diff
@@ -0,0 +1,6 @@
+package faaserrors
```

Comment:
The name of this file really should be `warnings.go` or `warn.go` since the only variable is not of error type.